.rubocop.yml

```diff
@@ -1,8 +1,9 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 3.0
```

Comment:
if you remove this line then rubocop will use `.ruby-version` (probably cleaner as we won't have to update this every time we bump ruby).